mathics_django/web/format.py

```diff
@@ -65,21 +79,16 @@ def format_output(obj, expr, format=None):
             result = expr.format(obj, "System`OutputForm")
         elif head == 'System`String':
             result = expr.format(obj, "System`InputForm")
+        elif head == 'System`Graphics3D':
```

Comment:
@rocky this won't, it depends in the value 400 be passed to the function `drawGraphics3d` (what MathML won't do).

To this works the parameter need be replaced by a constant.

Reply:
@rocky I replaced the parameter with a constant and the axes number bugged, then I remembered, the absolute size is relative to the container, not the canvas size. Fixing it now.

Reply:
@rocky I hadn't success, but I guess it'll work when MathML be removed of graphics 3d.

Reply:
Here is something save and will reduce the PR changes: reformat javascript according to the style mentioned in editorconfig. 


Which reminds me. Editorconfig should be changed so that it just applies to Javascript.  Python files should stay as they are and those are formatted by Python black.

Reply:
@rocky I'm formatting the JavaScript and CSS now.

Could you confirm the Python rules to me?
- indent style: spaces
- indent size: 4
- insert final newline: true

Reply:
Also, I guess we'll have to note skipping those css and js files such as for mathjax, and threejs that are not ours but are copied from elsewhere into our space.

Reply:
@rocky I'm doing it.

Also should the `three.js` file be deleted (we are using `three.min.js`)?

Reply:
@TiagoCavalcanteTrindade  : 
Install optional dependency used in Mathics Core [psutil](https://pypi.org/project/psutil/). We should also fix the python code so that it doesn't error out if psutil  or SystemMemory isn't available.